ds_judgements_public_ui/templates/includes/judgment_text_toolbar.html

```diff
@@ -1,11 +1,11 @@
-{% load i18n %}
+{% load static i18n %}
```

Comment:
What's the `static` for here? I can't see it being used in the changes below, but maybe i'm missing something?